app/screens/ARView.js

```diff
@@ -1,10 +1,10 @@
 import React, { Component, View } from 'react-native';
-import AR from './AR.js';
-import Map from './Map.js';
+import AR from '../components/AR';
```

Comment:
@bignell  making three smart view components for each view is very repetitive, especially since we'd next have to write filter mechanisms for which data we show(friends, savedpins..etc) in each of the render().  This may not be necessary. But for right now it works.  As a side note, we can improve this later by having one view container and using if statements to render AR/map/list.